app/cdap/components/SourceControlManagement/LocalPipelineListView/CommitModal.tsx

```diff
@@ -63,7 +64,7 @@ export const CommitModal = ({
       setPushLoadingMessage
     ).subscribe((res: any) => {
       // TODO: no changes made will also be 200, need to give different warning
-      if (res.statusCode !== 200) {
+      if (res.status !== SUPPORT.yes) {
```

Comment:
@sumengwang this change is required as https://github.com/cdapio/cdap-ui/pull/931 seems to have broken the banner.